examples/amp-fit-text.amp.html

```diff
@@ -0,0 +1,81 @@
+<!-- ## Introduction --><!--
+  [`amp-fit-text`](/content/amp-dev/documentation/components/reference/amp-fit-text-v0.1.md) enables you to manage the size and fit of text within a given area.
+--><!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+<!-- ## Setup -->
+    <!--
+      Include the `amp-fit-text` component.
+    -->
```

Comment:
These `<!-- ## -->` blocks are customarily elaborated for amp.dev code samples, but this file will not be ported to amp.dev. I'd recommend either adding this example in that repo or removing annotations from this file.